### ISSUE TITLE
[Proposal] Add Tutorials for MuJoCo based environments (contributors welcome)

### ISSUE THREAD

**Issue by Kallinteris-Andreas** [Farama-Foundation/Gymnasium#846]
### Proposal

With the release of the `MuJoCo-v5` environments, in Gymnasium 1.0.0 (which will be coming out, prior to the heat death of the universe).

We need tutorials on: 
- [ ] loading a quadruped custom robot model (WIP https://github.com/Farama-Foundation/Gymnasium/pull/838)
- [ ] loading a bipedal / humanoid custom robot model
- [ ] loading a custom scene e.g. [G-Barkour](https://github.com/google-deepmind/mujoco_menagerie/blob/main/google_barkour_v0/README.md) (which will have custom reward, obs?, terminal?)
- [ ] Using model with custom simulation options
- [ ] Create a Custom MuJoCo based environment class using `MuJoCoEnv`
- [ ] ??? Transfer learning across different models ???
- [ ] more tutorials ???

Contributors are welcome, with experience running RL on robotic environments, or with knowledge of the `MuJoCo simulator`, or with knowledge of Robotics, or just willingness to learn.
Comment here or discuss or join the gymnasium discord (invite link on the README) and ping me in the `# New Contributors`  channel

### Checklist

- [X] I have checked that there is no similar [issue](https://github.com/Farama-Foundation/Gymnasium/issues) in the repo

**Comment by jonzamora** [comment]
@Kallinteris-Andreas Is anyone working on `Create a Custom MuJoCo based environment class by subclassing MujocoEnv`? If not, I am working on *just* the right thing for this currently.

tldr; I am creating a MuJoCo version of [X-MAGICAL](https://github.com/kevinzakka/x-magical), which is originally created with a 2D simulator and shown below:

![shortstick-sweep](https://github.com/Farama-Foundation/Gymnasium/assets/50164418/f270c593-df71-415a-bf40-bf525bb3c2ed)


So far, my environment looks like this:


https://github.com/Farama-Foundation/Gymnasium/assets/50164418/b44ab71f-a0a7-4d41-bde3-9b6ed52d73ef


I'm integrating Gymnasium API support for the environment, and i'm doing this by subclassing `MujocoEnv`.

Let me know your thoughts, i'm happy to contribute this : )

**Comment by Kallinteris-Andreas** [comment]
@jonzamora feel free to submit a PR with your tutorial

(sorry for late reply, i though i had reply)

**Comment by ashwin-r11** [comment]
hi!, im interested on contribute by doing this  (loading a bipedal / humanoid custom robot model), is there anything i need to know before i start doing this? im new contributor btw. any words or advice is deeply appreciated.

**Comment by Kallinteris-Andreas** [comment]
@ashwin-r11 you can use the quadruped tutorial as reference, the more details you can include about your choice of environment parameters the better, include as much of the thought/experimental process for selecting some of those parameters the better 

when you create the PR please ping me

**Comment by ashwin-r11** [comment]
Ok done, once I do I'll ping you and raise a PR

**Comment by Lonny154** [comment]
Hi, I’m interested in contributing to this issue.

I’m currently working on reinforcement learning projects with Gymnasium and would like to add a beginner-friendly MuJoCo tutorial. I was thinking of starting with a focused tutorial on either:

1. Creating a simple custom MuJoCo environment by subclassing `MujocoEnv`
2. Using custom simulation options with an existing MuJoCo environment

Would either of these be useful as a first PR? I’d like to keep the scope small and avoid changing any environment behavior.

**Comment by Kallinteris-Andreas** [comment]
@Lonny154 contributions are welcome, do whichever you are most comfortable in doing

**Comment by DriftyPencil** [comment]
Hi,

I'm currently a Computing student at Imperial College London, currently researching AI Hallucination thresholds. 
I'd be interested in contributing the custom MujocoEnv tutorial. I've previously built custom MuJoCo environments for reinforcement-learning experiments, including defining observations and reward functions. Is this item still available, and are there any particular environment/design conventions you'd like the tutorial to follow?
